.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Your test.md is not supposed to be in your repository, make sure you double-check the requirements and fix it.